src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.body {
+  margin: 0;
+  padding: 0 8px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
I noticed that you're using a `flex-direction: column;` property for your `.search-bar` class, but I don't see any child elements inside this class in your CSS. If there are no child elements to arrange in a column, you don't need this property.